src/main/resources/db/migration/all/20240123104559__migrate_application_probation_ids_to_ap_area_ids.sql

```diff
@@ -0,0 +1,13 @@
+UPDATE
+    approved_premises_applications
+set
+    ap_area_id = (
+        SELECT
+            ap_area_id
+        from
+            probation_regions
+        where
+                id = approved_premises_applications.probation_region_id
+    )
+where
```

Comment:
i guess we want to drop the probation_region_id column at somepoint but understand why not now until happy wiht backfill ... shall we create a ticket to do this?

Reply:
I'll do it now, I'm going to move onto that next 👍